buildroot/package/systemd/systemd.mk

```diff
@@ -41,7 +41,8 @@ SYSTEMD_CONF_OPTS += \
 	-Dumount-path=/usr/bin/umount \
 	-Dnobody-group=nogroup \
 	-Didn=true \
-	-Dnss-systemd=true
+	-Dnss-systemd=true \
+	-Dapparmor=false
```

Comment:
This change also needs to be a patch in buildroot-patches/ so we can update buildroot with scripts.

And ideally, also sent upstream to buildroot :wink:

Reply:
Okay I will just remove this for now.